engine/crates/composition/tests/composition/override_basic/federated.graphql

```diff
@@ -23,9 +23,9 @@ enum join__Graph {
 
 type Mammoth {
     tuskLength: Int
-    weightGrams: Int @join__field(graph: mangrove, overrides: "steppe")
+    weightGrams: Int @join__field(graph: mangrove, override: "steppe")
```

Comment:
We were rendering that wrong. You'll see in from_sdl() that we still accept "overrides" for backward compatibility, but "override" is correct.